src/utils/notification.ts

```diff
@@ -4,15 +4,15 @@ const notification = Toaster.create({
   position: Position.TOP
 })
 
-export const showSuccessMessage = (message: string, timeout = 500) => {
+export const showSuccessMessage = (message: string, timeout: number) => {
   notification.show({
     intent: Intent.SUCCESS,
     message,
     timeout
   })
 }
 
-export const showWarningMessage = (message: string, timeout = 500) => {
```

Comment:
As discussed, could you try `timeout: number = 500`? If it doesn't work, then keeping the current change is fine too